Podfile

```diff
@@ -3,15 +3,29 @@ xcodeproj 'malaria-ios.xcodeproj'
 
 platform :ios, '8.0'
 use_frameworks!
-link_with 'malaria-ios', 'malaria-iosTests'
 
-pod 'Alamofire', '~> 3.1.4'
-pod 'SwiftyJSON', '~> 2.3.1'
-pod 'Charts', '~> 2.1.6'
-pod 'CVCalendar', '~> 1.2.8'
+def testing_pods
```

Comment:
Change name, "testing-pods" is not suitable unless it really is. testing pods would make more sense to target malaria-iosTests.

Moreover, Charts, CVCalendar and the remaining UI views can be on a different `def test-dependencies` since they are not required for unit testing.